modules/bitgo/test/unit/bitgo.ts

```diff
@@ -609,4 +609,42 @@ describe('BitGo Prototype Methods', function () {
       );
     });
   });
+
+  describe('authenticate', function () {
+    it('should get or create the ecdhKeychain if ensureEcdhKeychain is set', async function () {
```

Comment:
Please make these two separate tests:

`should get ecdhKeychain if already set`

`should create the ecdhKeychain if not already is set`